docs/_redirects

```diff
@@ -1,330 +1,344 @@
+https://docs.directus.io/blog/building-a-personal-travel-journal-with-vue-js-and-directus.html                              https://directus.io/docs/tutorials
+https://docs.directus.io/blog/passwordless-sms-authentication-with-plivo-and-directus-automate.html                         https://directus.io/docs/tutorials
+https://docs.directus.io/blog/build-a-url-shortener-with-react-type-script-and-directus.html                                https://directus.io/docs/tutorials
+https://docs.directus.io/blog/building-a-testimonial-widget-with-sveltekit-and-directus.html                                https://directus.io/docs/tutorials/projects/build-a-testimonial-widget-with-sveltekit-and-directus
+https://docs.directus.io/blog/building-a-job-board-platform-with-directus-and-solid-start-js.html                           https://directus.io/docs/tutorials
+https://docs.directus.io/blog/building-a-hotel-booking-system-with-directus-next-js-and-stripe.html                         https://directus.io/docs/tutorials/projects/build-an-hotel-booking-platform-with-next-js-stripe-and-directus-automate
+https://docs.directus.io/blog/building-a-video-streaming-app-with-sveltekit-and-directus.html                               https://directus.io/docs/tutorials/projects/build-a-video-streaming-app-with-sveltekit-and-directus
+https://docs.directus.io/blog/getting-started-with-directus-and-laravel.html                                                https://directus.io/docs/tutorials/getting-started/fetch-data-from-directus-with-laravel
+https://docs.directus.io/blog/getting-started-directus-ios.html                                                             https://directus.io/docs/tutorials/getting-started/fetch-data-from-directus-in-ios-with-swift
+https://docs.directus.io/blog/using-directus-auth-with-ios.html                                                             https://directus.io/docs/tutorials/getting-started/implement-directus-auth-with-ios
+https://docs.directus.io/blog/deploy-directus-ubuntu-server.html                                                            https://directus.io/docs/tutorials/self-hosting/deploy-directus-to-an-ubuntu-server
+https://docs.directus.io/blog/deploying-directus-to-aws-ec2-with-docker.html                                                https://directus.io/docs/tutorials/self-hosting/deploy-directus-to-aws-ec2
+https://docs.directus.io/blog/deploying-directus-to-google-cloud-platform-with-docker.html                                  https://directus.io/docs/tutorials/self-hosting/deploy-directus-to-google-cloud-platform
+https://docs.directus.io/blog/implement-directus-auth-in-next-js-14.html                                                    https://directus.io/docs/tutorials/getting-started/implement-directus-auth-with-next-js
```

Comment:
This looks like another reference to a page which is giving me a 404 <https://directus.io/docs/tutorials/getting-started/implement-directus-auth-with-next-js>

Reply:
```suggestion
https://docs.directus.io/blog/implement-directus-auth-in-next-js-14.html                                                    https://directus.io/docs/tutorials/getting-started/using-authentication-in-next-js
```